docs/automated_testing.md

```diff
@@ -991,38 +993,162 @@ uses: suffolkLITLab/ALKiln@releases/v4
 
 When you want to update to a new version of the ALKiln, update that sha manually.
 
+### Set ALKiln's npm version
+
+This one requires prior technical knowledge. Feel free to ask us.
+
+You can use an exact npm version of ALKiln by using your workflow file's `ALKILN_VERSION` input. The default uses a carat, for example `^4.0.0`. That means it will use the latest minor or patch in version 4 of ALKiln. You can instead use a specific version, for example `4.3.0`. See our section on [setting optional inputs](#optional-inputs).
```

Comment:
```suggestion
You can use an exact npm version of ALKiln by using your workflow file's `ALKILN_VERSION` input. The default uses a carat, for example `^4.0.0`. That means it will use the latest minor or patch in version 4 of ALKiln. You can instead use an exact version, for example `4.3.0`. See our section on [setting optional inputs](#optional-inputs).
```